### PR TITLE
Add 'name' field docs & minor fixes

### PR DESCRIPTION
**Pull request by vansosnin**
Added `name` field docs. I found it accidentally in the error message. And this is exactly what I needed in my work 🙂
Also added missing field types for sibling fields.

### CODE REVIEW

File: src/content/configuration/other-options.md [hunk 1]
```diff
@@ -146,3 +153,17 @@ module.exports = {
...
 ```
+
+
+## `name`
+
+`string`
+
+Name of the configuration. Used when loading multiple configurations.
+
+```js
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]: use `javascript` vs `js`

File: src/content/configuration/other-options.md [hunk 2]
```diff
@@ -146,3 +153,17 @@ module.exports = {
   recordsOutputPath: path.join(__dirname, 'newRecords.json')
 };
 ```
+
+
+## `name`
+
+`string`
+
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]: please also add `__webpack.config.js__` above the code blocks

Comment (vansosnin) [reply src/content/configuration/other-options.md]: Both review fixes done. Also fixed inconsistency in other examples on this page.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**